app/views/Dashboard/styles.css

```diff
@@ -0,0 +1,34 @@
+.dashboard-navigation {
+    background-color: var(--color-foreground);
+
+    .dashboard-buttons {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding: var(--dui-spacing-small);
+        gap: var(--dui-spacing-medium);
+    }
```

Comment:
Instead of overriding button style, check if we can use already available button prop

Reply:
This is a button container whose name will be renamed to "dashboard-buttons-wrapper"